src/run.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const logger = require('winston');
+const parser = require('js-yaml');
+const fs = require('fs');
+const _ = require('lodash');
+
+let spawnSync = require('child_process').spawnSync;
+
+function expandTokens(command, vars) {
```

Comment:
Personally, I like that function is an obvious key. => {} is typically used with anonymous functions, so why give it a name? Also, it changes the value of the this object, which might confuse some people who aren't aware that "this" in a function is different from "this" in a lambda.